src/js/themes/hpe.js

```diff
@@ -81,7 +81,7 @@ export const hpe = deepFreeze({
     input: {
       font: {
         height: 'inherit',
-        weight: 500,
+        weight: 450, // weight: 500,
```

Comment:
I didn't see 450 in the list of font weights we're asking for in the CDN.

Reply:
I believe this is a temp fix related to how the font-family and weight definitions needed to be done to allow 300 to be default weight. Matt mentioned he might have to fudge some things on this PR but that any grommet enhancements would be done once the typography direction is nailed dow.

Reply:
Yes, this is a temporary hack for preview purposes only.

The typography direction calls for Metric-light to be the new default. However, Grommet Text does not currently allow weight to be defined by the theme (only family).

For expediency, I'm proposing we use this configuration for assessing the typography styling impacts and cataloging the needed work to support.